examples/funloc/analysis_fun.py

```diff
@@ -52,7 +52,7 @@
 
 # Set parameters for remotely connecting to acquisition computer
 params.acq_ssh = 'minea'  # Could also be e.g., "eric@minea.ilabs.uw.edu"
-params.acq_dir = '/net/sinuhe/data02/eric_non_space'
+params.acq_dir = 'eric_non_space' # name of remote dir containing data
```

Comment:
So the change would be that `params.acq_dir` can now be a `str` or `list of str`